### DISCUSSION TITLE
How to insert a formula field in the form?

### DISCUSSION THREAD

**Discussion by rcarvalho2clicks** [Budibase/budibase#11248] [Help] [answered]
Guys, I need to include the formula field in the form and I didn't find how to do that.
![2023-07-17_07-43](https://github.com/Budibase/budibase/assets/135126750/0add50a7-d3d0-4180-80c4-16ae868cf59d)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @rcarvalhopiaget,

The short answer you can't, these are fields which you can't submit to. 

They are mainly used to combine other fields together within the data area; e.g. last name and first name to become full name which can then be outputted in the design area as `full name`. Another example would be using a math based function to add up the total price of `A B and C` columns. This can then be used in the design area in a range of different ways as well.

Depending on the type of formula you use you can also use them for filtering.

I hope this makes sense.

**Comment by rcarvalho2clicks** [comment]
Thank you for the clarification.

Em seg., 17 de jul. de 2023 às 09:16, Conor Webb ***@***.***>
escreveu:

> Hey @rcarvalhopiaget <https://github.com/rcarvalhopiaget>,
>
> The short answer you can't, these are fields which you can't submit to.
>
> They are mainly used to combine other fields together within the data
> area; e.g. last name and first name to become full name which can then be
> outputted in the design area as full name. Another example would be using
> a math based function to add up the total price of A B and C columns.
> This can then be used in the design area in a range of different ways as
> well.
>
> Depending on the type of formula you use you can also use them for
> filtering.
>
> I hope this makes sense.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/11248#discussioncomment-6467088>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAG55XXIWYTRFVBKAE433A3XQUUJHANCNFSM6AAAAAA2MYCPKI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
<http://www.jpiaget.com.br> *Rodrigo Damasceno de Carvalho*
Supervisor de Tecnologia da Informação

(11) 4367-8303 / (11) 99124-9874
***@***.***
